obspy/clients/iris/tests/test_client.py

```diff
@@ -58,14 +58,18 @@ def test_distaz(self):
         client = Client()
         # normal request
         result = client.distaz(stalat=1.1, stalon=1.2, evtlat=3.2, evtlon=1.4)
-        self.assertAlmostEqual(result['distance'], 2.09554)
-        self.assertAlmostEqual(result['backazimuth'], 5.46946)
-        self.assertAlmostEqual(result['azimuth'], 185.47692)
+        self.assertAlmostEqual(result['distance'], 2.10256)
```

Comment:
Is there a reason for the change in results? Just more correct? The new one seems to match geographiclib at least.